astrolabe/cli.py

```diff
@@ -506,5 +536,11 @@ def validate_workload_executor(workload_executor, startup_time,
         exit(1)
 
 
+@spec_tests.command('stats')
```

Comment:
This can simply be `@spec_tests.command()` . You only need to include the name of the command in the call to `command()` if it differs from the name of the method being decorated.

Reply:
Alright, changed.